### PR TITLE
UI Test for button events

### PR DESCRIPTION
**Pull request by carldebilly**
# Additional PR
This is UI tests missing for the previous PR #1316

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Test_Button_Click.cs [hunk 1]
```diff
@@ -89,5 +90,134 @@ void CheckClickCounts(int layer1Count, int layer1InnerCount, int layer2Count, in
...
+			_app.DragCoordinates(
+				fromX: xVisible,
+				fromY: btnTappedRect.CenterY,
+				toX: btnTappedRect.Right - 1,
+				toY: btnTappedRect.CenterY
+			);
+			CheckCount(btnTappedCount, 2, msg: "after dragged tapped (should not tap)");
+
+			// ---- DOUBLE TAPPED ----
+			/*CheckCount(btnDoubleTappedCount, 0, "double tapped starting value");
```
Comment (jeromelaban) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Test_Button_Click.cs]: Is there an issue for this disabled test ?

Comment (carldebilly) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Test_Button_Click.cs]: Yes...  double tap not supported yet by all platforms on Uno.

Comment (carldebilly) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Test_Button_Click.cs]: I hope this code will be uncommented by David's PR on touches...

Comment (dr1rrb) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Test_Button_Click.cs]: #1256

**Pull request merged**